templates/zerver/integrations/errbot.md

```diff
@@ -25,23 +29,38 @@ Run your favorite chatbot in Zulip!
         BOT_LOG_FILE = r'<path/to/your/errbot/logfile.log>'
         BOT_LOG_LEVEL = logging.INFO
 
-        BOT_IDENTITY = {  # Fill this with the corresponding values in your bot's `.zuliprc`
-          'email': '<err-bot@your.zulip.server>',
+        BOT_IDENTITY = {  # Use the corresponding values in your bot's `.zuliprc`
+          'email': '<your-bot-name@your_organization.zulipchat.com>',
           'key': '<abcdefghijklmnopqrstuvwxyz123456>',
-          'site': '<http://your.zulip.server>'
```

Comment:
Previously, this entire code block wasn't edited, to remain in sync with the repo code.
But, it has now been edited in several places as per recommendations, as this is user-facing documentation and it's fine to not be in sync with the repo.